iocage/lib/ioc_fetch.py

```diff
@@ -171,18 +174,23 @@ def fetch_release(self, _list=False):
             except OSError as err:
                 raise RuntimeError(f"{err}")
 
-            if os.path.isdir(f"{self.iocroot}/download/{self.release}"):
+            dataset = f"{self.iocroot}/download/{self.release}"
+
+            if os.path.isdir(dataset):
                 pass
             else:
-                su.Popen(["zfs", "create", "-o", "compression=lz4",
-                          f"{self.pool}/iocage/download/"
-                          f"{self.release}"]).communicate()
-            dataset = f"{self.iocroot}/download/{self.release}"
+                self.zpool.create(f"{self.pool}/iocage/download/{self.release}", {
+                        "compression": "lz4"
+                    });
```

Comment:
I've written too much JavaScript - putting semicolons wherever I can now. Should see a doctor if this happens more often.